app/views/provider_interface/candidate_pool/shares/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :browser_title, t('.title') %>
+<% content_for :before_content, govuk_back_link_to(@back_link) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render ServiceInformationBanner.new(namespace: :provider) %>
+
+    <span class="govuk-caption-xl"><%= t('.candidate_number', candidate_id: @candidate.id) %></span>
+    <h1 class="govuk-heading-xl"><%= @application_form.redacted_full_name %></h1>
+
+    <p class='govuk-body'><%= t('.copy_and_paste_message') %></p>
+    <p class='govuk-body govuk-!-font-weight-bold'><%= @back_link %></p>
```

Comment:
I think we should strip out the return_to params, so this is just the link to the candidate's profile.